app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientView.kt

```diff
@@ -163,4 +163,6 @@ class LinkIdWithPatientView(
         endAction = runAfter
     )
   }
+
+  fun isVisible(): Boolean = (visibility == View.VISIBLE)
```

Comment:
This seems like a good candidate for an extension function for the `View` class. What do you think of moving this to `Views.kt` and making it an extension function?

Reply:
So the reason I avoided it is, if we use androidx `core-ktx` package we can get several extension functions for view. Instead of trying to reimplement those I was hoping we can add it maybe. wdyt?